qa-test-app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
This file should also add this line
`src/main/res/values/mapbox_access_token.xml` since this file will be added to this module